MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -35,17 +37,24 @@ It is also important to note that while this document provides the criteria agai
 # Criteria
 
 ## Frontend
-* If provided, End-to-end tests must be written in an [officially approved technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) (3, 4)
+
+Note: Frontend criteria apply to both modules and shared libraries. 
+
+* If provided, End-to-end tests must be written in an [officially supported technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) (3, 4)
   * -_note: while it's strongly recommended that modules implement integration tests, it's not a requirement_
   * -_note: these tests are defined in https://github.com/folio-org/stripes-testing_
 * Have i18n support via react-intl and an en.json file with English texts (8)
 * Have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension (9)
 * Use the latest release of Stripes at the time of evaluation (10, 16)
 * Follow relevant existing UI layouts, patterns and norms (10) -_note: read more about current practices at [https://ux.folio.org/docs/all-guidelines/](https://ux.folio.org/docs/all-guidelines/)_
   * E.g. Saving state when navigating between apps (or confirming that you'll lose the state)
+  * For UI links to documentation, there is no rule on where that documentation should be hosted, i.e. docs.folio.org, or wiki.folio.org, or module-specific destinations, as long as it is publicly accessible.
 * Must work in the latest version of Chrome (the supported runtime environment) at the time of evaluation (10)
 
 ## Backend
+
+Note: Backend criteria apply to modules, shared backend libraries, and edge modules.
```

Comment:
No one on the subgroup or the #tech-council slack was able to offer any specific criteria that should be different for either edge modules or shared backend libraries.  We've never had any submission of a backend library, and the only person to submit an edge module (@vijay-eis) is not available to offer suggestions.

If this actually comes up as a problem during an evaluation, we now have a lightweight process (in this PR) to deal with it.  Per the process doc

> If the TC determines that some failed criteria would be resolved by non-controversial changes to the criteria themselves (or referenced requirements like the Officially Supported Technologies), TC may decide to accept the module and make the agreed-upon changes.

Reply:
It might be a little _too much_ wiggle room, but an "as appropriate" at the end of the sentence could also provide a bit of an escape hatch in cases where the different types of backend-related projects don't really line up. Like you said, though, that might not be necessary anymore

Reply:
I see that point @mweaver-ebsco.  Personally I would prefer to identify whatever situations exist where the criteria might not be appropriate, so we can actually adjust them.  Because I think that fits better with the general goal of the criteria themselves (and the evaluation) being as objective as possible, while the decision can take things like that into account.